pkl-core/src/main/java/org/pkl/core/stdlib/test/report/SimpleReport.java

```diff
@@ -17,19 +17,32 @@
 
 import java.io.IOException;
 import java.io.Writer;
+import java.util.List;
 import java.util.stream.Collectors;
 import org.pkl.core.TestResults;
 import org.pkl.core.TestResults.TestResult;
 import org.pkl.core.TestResults.TestSectionResults;
+import org.pkl.core.runtime.AnsiCodingStringBuilder;
+import org.pkl.core.runtime.AnsiCodingStringBuilder.AnsiCode;
+import org.pkl.core.util.ColorTheme;
 import org.pkl.core.util.StringUtils;
 
 public final class SimpleReport implements TestReport {
 
+  private static final String passingMark = "✔ ";
+  private static final String failingMark = "✘ ";
```

Comment:
Why the naming style deviation? If for readability (which I tend to agree with, because I'm alright with my tooling telling me something is `static final`), might we also convert the members of `ColorTheme` - which are not `final` - to camelCase?